src/ui/js/Env.js

```diff
@@ -625,4 +662,4 @@ Env.validateUrl = function(url) {
   // This also doesn't verify that the URL points to a safe page, but that is
   // outside of the scope of this function.
   return url;
-};
+};
```

Comment:
Looks like you lost the trailing whitespace on Env.js.  If i'm reading the diff right, please fix that.